Makefile

```diff
@@ -417,8 +419,9 @@ $(BUILD_DIR)/baserom/%.o: $(BASEROM_SEGMENTS_DIR)/%
 $(BUILD_DIR)/data/%.o: data/%.s
 	$(AS) $(ASFLAGS) $< -o $@
 
-$(BUILD_DIR)/assets/text/%.enc.h: assets/text/%.h assets/text/charmap.txt
-	$(PYTHON) tools/msgenc.py assets/text/charmap.txt $< $@
+$(BUILD_DIR)/assets/text/%.enc.h: assets/text/%.h $(EXTRACTED_DIR)/text/%.h assets/text/charmap.txt
```

Comment:
The lack of symmetry between `$(BUILD_DIR)/assets/text/%.enc.h` and `$(EXTRACTED_DIR)/assets/text/%.enc.h` bugs me so I started a discord discussion about it, but I guess we don't have to do anything for now